### ISSUE TITLE
desktop capture as texture

### ISSUE THREAD

**Issue by Elmapul** [Sean-Bradley/Webcam-As-Texture-in-Blender#1]
its possible to capture the desktop (or some windows from it) to use it as an texture?

**Comment by Sean-Bradley** [comment]
I can't do it myself yet.
But if you can program c++ on windows, you could try using directshow to intercept the desktop region and output it as webcam. There are also drivers you can use to do it, but I can't name any good ones.
Probably search for 'desktop as webcam driver'.

**Comment by Elmapul** [comment]
i'm on linux.
i guess python has an solution for it (i dont know if  python have,  but obs has)

**Comment by Sean-Bradley** [comment]
I use obs for this kind of thing now, look out my video tutorial 
https://www.youtube.com/watch?v=joh--ToDY7c

**Comment by Elmapul** [comment]
no, no, obs allow you to crop with chroma key and do picture in picture, but it dont allow to do 3D effects with the image of the camera

**Comment by Sean-Bradley** [comment]
I created this video to show how to view webcam on 3d object inside OBS
https://www.youtube.com/watch?v=SdYf4OvnVxY

project at
https://github.com/Sean-Bradley/Webcam-as-Texture-using-Three.js

I havn't figured out desktop recording just yet.
